design/###-tools-visualstudio.md

```diff
@@ -0,0 +1,46 @@
+
+# Proposal: Tools - Visual Studio 2017 (15.2) required
+
+| **Status**        | **Proposed/Accepted/Deprecated** |
+|:------------------|:---------------------------------------------|
+| **RFC #**         | ####                                         |
+| **Submitted**     | YYYY-MM-DD                                   |
+| **Dependencies**  | RFC #, #                                     |
+
+---
+
+## Summary
+Declare **Microsoft Visual Studio 2017 (15.2)** as the minimal version supported.
+
+
+## Motivation
+Visual Studio 2017 (15.2) was [released on May 2017](https://docs.microsoft.com/visualstudio/releasenotes/vs2017-relnotes-v15.2). This version allows to install
+side-by-side different versions of the MSVC toolset, it allows you to [build binaries
+that are fully compatible with the ones built using older versions](https://devblogs.microsoft.com/cppblog/stuck-on-an-older-toolset-version-move-to-visual-studio-2015-without-upgrading-your-toolset/).
+
+Visual Studio 2017 (15.2) includes [`wswhere` application](https://github.com/microsoft/vswhere). 
```

Comment:
I would like to fix that type here, `wswhere` -> `vswhere`
but the repo jgsogo:proposal does not exist (for me)  , so I do not know how or where to submit the pr for the pr :-)

Reply:
Now the repository should be public